### ISSUE TITLE
enhancement: allow to import agenda points as subitem

### ISSUE THREAD

**Issue by donovaly** [OpenSlides/OpenSlides#3086]
Take for example this agenda:

1. Begrüßung
2. Wahl der Mandatsprüfungs- und der Zählkommission
3. Rechenschaftsbericht
   a. Bericht des Vorsitzenden
   b. Bericht des Kassierers
   c. Bericht der Revisoren
   d. Aussprache zu den Berichten
4. Entlastung des Kassierers

It is at the moment impossible to import such an agenda so that the points a.- d. are imported as subitem of point 3..

**Comment by tsiegleauq** [comment]
I don't really get the idea behind this.

1. One cannot import an agenda. Importing an agenda would not make much sense.
2. Sorting the agenda already allows to create subitems

**Comment by donovaly** [comment]
> I don't really get the idea behind this.

The idea is to import an agenda as in my initial post to this report.
If such an agenda can now correctly be imported with OS 3, then you can close this report.

**Comment by tsiegleauq** [comment]
As mentioned before:
> One cannot import an agenda. Importing an agenda would not make much sense.

The core problem is, that the agenda is not a specific object. The agenda represents the overall structure of: Whatever you want to do at your gathering.
What you are usually doing at a gathering is, at least from out perspective: Topics, Motions and Assignments.
Currently, you can import motions from CSV (although that feature is a little shabby). You cannot import topics nor assignments - would be possible, but the demand was not there yet.

What I am getting from your feature request is, that you want to upload a file (whatever structure that might have), that executes following tasks for you:

> 1. Begrüßung
create a topic, name it "Begrüsung", give it agenda item weight 1
> 2. Wahl der Mandatsprüfungs- und der Zählkommission
create an election, name it so, give it agenda item weight 2
> 3. Rechenschaftsbericht
create a motion (I guess), name it so, give it agenda item weight 3
> a. Bericht des Vorsitzenden
I don't even get what that's supposed to be
> 4. Entlastung des Kassierers
Create a motion, name it so [ ... ]

**Comment by donovaly** [comment]
> What I am getting from your feature request is, that you want to upload a file (whatever structure that might have), that executes following tasks for you

Yes, but not that sophisticated. It should just create the agenda points, like for "Begrüßung" (meaning "Welcome words"). If an agenda point will be a motion, an election etc. is something I want to set up later.

It is an enhancement request not a bug report. Please decide if you want to implement it or not. If not, close the report, if yes, please don't assign it to me since I am no developer.

**Comment by tsiegleauq** [comment]
an assigned Issue does not necessarily mean that you are supposed to commit code ;-)

That feature request is a little more sophisticated and requires some more conceptual work. I leave that open for now in case somebody likes to pick it up

**Comment by Elblinator** [comment]
Something similar is in discussion and hence this issue will be closed and the corresponding issue will get it's own issues in the responsible repositories
